components/raftstore-v2/src/operation/query/local.rs

```diff
@@ -96,7 +271,7 @@ where
 }
 
 #[derive(Clone)]
-struct StoreMetaDelegate<E>
+pub struct StoreMetaDelegate<E>
```

Comment:
It's only used in this file, doesn't need to be public.

Reply:
It's also used in router/imp.rs